guestbook.py

```diff
@@ -94,7 +105,9 @@ def post(self):
         greeting = Greeting(parent=guestbook_key(guestbook_name))
 
         if users.get_current_user():
-            greeting.author = users.get_current_user()
+            author = Author(identity=users.get_current_user().user_id(),
```

Comment:
maybe just greeting.author = ...

Or you could add a static method to Author and do greeting.author = Author.fromUser(user).

Or leave as is, just wanted to put out other options.